contrib/debian-template/rules

```diff
@@ -17,6 +18,7 @@ override_dh_auto_configure:
 	dh_auto_configure -- \
 		--with-libfabric=/opt/amazon/efa \
 		--prefix=/opt/amazon/ofi-nccl \
+		--libdir=/opt/amazon/ofi-nccl/lib \
```

Comment:
This should really be defined in terms of prefix, but if this is how other efa packages are doing it, I'm fine with this.